README.md

```diff
@@ -22,10 +22,31 @@
   subdirectory.
 
 
+## Configuring tests
```

Comment:
I would rather **always** have the `config.yml` (possibly better name `menu.yml`) and nothing left to default. It simplifies the logic downstream. Upstream (_i.e._ `.travisy.yml` and `.appveyor.yml`) we can install all dependencies without logical switches on environment variables.

Reply:
No problem to have this mandatory but would prefer to have this outside this PR. However I will rename to `menu.yml` and append to this PR.

Reply:
Mandatory `menu.yml` follewed up in #36.

Reply:
Fine by me. One step at a time.